modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/DetectorMappingController.java

```diff
@@ -89,4 +90,11 @@ public MatchingDetectorsResponse searchDetectorMapping(@RequestBody List<Map<Str
         MatchingDetectorsResponse matchingDetectorMappings = detectorMappingRepo.findMatchingDetectorMappings(tagsList);
         return matchingDetectorMappings;
     }
+
+    @RequestMapping(value="/search/findByDetector", method = RequestMethod.DELETE)
+    @ResponseStatus(HttpStatus.OK)
+    public void deleteMappingByDetectorUUID(@RequestParam UUID uuid){
```

Comment:
same here. deleteMappingsByDetectorUUID

Reply:
I have updated the PR, with all review fixes included